static/sitemap/PriorityPlaces.0.txt

```diff
@@ -0,0 +1,151 @@
+https://datacommons.org/place/geoId/01
```

Comment:
The states are already covered in State.0.txt, I wonder if we need a duplicated sitemap, or could that be an issue for search engine.

Reply:
From what I understand having the duplicates shouldn't break anything, but also won't increase crawl rate (having the link twice will not make googlebot crawl it twice).

Having a dedicated sitemap is still helpful for us because the recommended way to request re-crawling for a batch of URLs is to put them in a sitemap. Otherwise, we'd have to manually request re-crawling for each of the 100 top cities. (The cities sitemap has 45,000+ entries, more than we would want for a test set.)

Reply:
That makes sense.